yt_dlp/extractor/espn.py

```diff
@@ -49,6 +55,8 @@ class ESPNIE(OnceIE):
             'description': 'md5:39370c2e016cb4ecf498ffe75bef7f0f',
             'timestamp': 1390936111,
             'upload_date': '20140128',
+            'duration': 1302,
+            'thumbnail': 'md5:328b04abedca5cc2a55d76d613759de1',
```

Comment:
maybe better to use a regex instead

Reply:
That's an old test that I just updated to get it to pass, should I still make adjustments?

Reply:
I mean for the thumbnail. Eg: 
```suggestion
            'thumbnail': r're:https://.+\.jpg',
```